### DISCUSSION TITLE
disabling WYSIWYG

### DISCUSSION THREAD

**Discussion by graphis** [GetPublii/Publii#1523] [Help]
Hello, 
is there a way to disable the WYSIWYG editor? 
thanks

**Comment by candidexmedia** [comment]
Any reason why you want to disable it?

**Reply by graphis** [reply]
yes, I simply don't want one of my users for example to format bold italic parts in a paragraph, or no inline images inside paragraphs, etc.

**Comment by dziudek** [comment]
Hi,

Please consider using Markdown editor - you can use the HTML directly in the post content then

**Reply by graphis** [reply]
Hi, ok, just to clarify.

I clearly can choose another option if I don't want to use the WYSIWYG editor. However, there are other people in the world for whom the options in the WYSIWYG editor are a distraction and who are not able / or don't want to / to control the formatting they use. Some of these people are among my users. 
I spend a lot of time creating themes for Publii, and I simply don't want the WYSIWYG editor to interfere with my design. 
I just wanted to ask if there is an option for this. I completely understand if that is not the case.

thanks

**Reply by dziudek** [reply]
As I wrote - you can use Markdown editor - it has no UI and allows users to put any HTML in the post content. It is the simplest solution to not use WYSIWYG editor, during editing posts in Publii.

Publii has three editors and every editor is for different needs - in my opinion use of Markdown editor instead of WYSIWYG editor is for your case

**Reply by graphis** [reply]
ok thanks you have convinced me :D I will use from now on the Markdown editor
